### ISSUE TITLE
cb is not a function on 1.0.0-beta.0

### ISSUE THREAD

**Issue by scttcper** [getsentry/raven-node#237]
I had a small koa project's tests explode on 1.0.0-beta.0 even though the dsn was set to false for testing. The main error being `Uncaught TypeError: cb is not a function`. Any help would be appreciated.

travis: https://travis-ci.org/scttcper/honbot-server/builds/177569614
setup: https://github.com/scttcper/honbot-server/blob/8ffd67afe56c2d3e67d5a1461578925159b160c6/src/index.js#L23-L31

**Comment by scttcper** [comment]
I've trimmed it down.
npm i koa@next raven@beta

```js
const Koa = require('koa');
const Raven = require('raven');

const app = new Koa();
const client = app.context.raven = Raven
  .config(false)
  .install({ unhandledRejection: true });

app.on('error', (err, ctx) => {
  const parsedReq = client.parsers.parseRequest(ctx.request);
  client.captureException(err, parsedReq);
});

app.use(() => {
  throw new Error('Terrible Error');
});

app.listen(3000);
```

result is infinite loop
```
raven@1.0.0-beta.0 alert: uncaughtException: ddaa15444bcd4ad9ac6338aef4d3343b
raven@1.0.0-beta.0 alert: uncaughtException: 22c8a18141784336b6d76b8f06386af2
raven@1.0.0-beta.0 alert: uncaughtException: c58dbf54f26342c78592dc284ddc5049
```

**Comment by LewisJEllis** [comment]
Uh-oh - I think I know why `cb is not a function` happened, but I'm not sure about this loop. I'm wondering if errors in Koa result in both a process `unhandledRejection` and a koa app `error` event being emitted and things get weird from there. I haven't really exercised the beta with anything but express and simple toy examples, so this is really helpful. I'll take your trimmed example and investigate.

**Comment by LewisJEllis** [comment]
@scttcper I published 1.0.0-beta.1 which should resolve this. Turned out it was just the "DSN disabled" case not being defensive enough. You should be able to install it with `npm install raven@beta` or `npm install raven@1.0.0-beta.1`.

**Comment by scttcper** [comment]
great working as expected now
